probe-rs/targets/nRF91_Series.yaml

```diff
@@ -5,38 +5,67 @@ manufacturer:
 chip_detection:
 - !NordicFicrInfo
   part_address: 0xff0140
-  variant_address: 0xff021c
+  variant_address: 0xff0148
   part: 0x9160
   variants:
-    0x400: nRF9160_xxAA
+    0x41434953: nRF9160_xxAA
+- !NordicFicrInfo
+  part_address: 0xff0140
+  variant_address: 0xff0148
+  part: 0x9151
+  variants:
+    0x4143414c: nRF9120_xxAA
```

Comment:
At least add a comment that this isn't a typo

Reply:
I'm not sure about redundant, I think probe-rs is supposed to match the `part` value so it would not have recognised a 9161 🤔 Or if it did, there might be a bug in the detection code. Thanks for the change, and sorry for forgetting about this PR